docs/ocis/development/continuous-integration.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Continuous Integration"
+date: 2020-10-01T20:35:00+01:00
+weight: 100
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/development
+geekdocFilePath: continuous-integration.md
+---
+
+{{< toc >}}
+
+oCIS uses [DRONE](https://www.drone.io/) as CI system. You can find the pipeline logs [here](https://drone.owncloud.com/owncloud/ocis) or in your PR.
+
+## Concepts
+
+The pipeline is defined in [Starlark](https://github.com/bazelbuild/starlark) and transformed to YAML upon pipeline run. This enables us to do a highly dynamic and non repeating pipeline configuration.
+
+Upon running the pipeline, your branch gets merged to the master branch. This ensures that we always test your changeset if as it was applied to the master of oCIS. Please note that this does not apply to the pipeline definition (`.drone.star`).
+
+## Things done in CI
+
+- static code analysis
+- linting
+- running UI tests
+- running ownCloud 10 test suite against oCIS
+- build and release docker images
+- build and release binaries
+- build and release documentation
+
+## Flags in commit message and PR title
+
+You may add flags to your commit message or PR title in order to speed up pipeline runs and take load from the CI runners.
+
+- `[CI SKIP]`: no ci is run on the commit or PR
+
+- `[docs-only]`: please add this flag, if you only changed documentation. This will only trigger documentation related CI steps.
```

Comment:
```suggestion
- `[docs-only]`: please add this flag, if you only changed documentation. This will only trigger documentation related CI steps.

- `[tests-only]`: please add this flag, if you only changed tests or test-related tooling. You do not need to add a changelog for tests-only changes.
```